sshr/logger.go

```diff
@@ -0,0 +1,16 @@
+package sshr
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+)
+
+type logger struct {
+	user string
+}
+
+func (l *logger) info(format string, args ...interface{}) {
```

Comment:
Hmm, 
I think using `infof` instead of `info` for this function name would be better, as it suggests that it accepts formatting.

Reply:
I fix it in commit https://github.com/heat1024/sshr/pull/1/commits/28b51562039ee27217c01d1aeb58184986e26edf
Thx!